tools/mo/install_prerequisites/install_prerequisites.sh

```diff
@@ -28,7 +28,7 @@ for ((i=1;i <= $#;i++)) {
         *)
             if [[ "$1" != "" ]]; then
                 echo "\"${!i}\" is unsupported parameter"
-                echo $"Usage: $0 {caffe|tf|tf2|mxnet|kaldi|onnx} {venv}"
```

Comment:
I believe we should keep `tf2` option in order to escape confusion for those users are used to using it in previous versions.

Reply:
Moved back.